### PR TITLE
{DJAN-4} Setup Github

### PR DESCRIPTION
**Pull request by vkurup**
Instructions on how to set up Github.

### CODE REVIEW

File: source/services/github.rst [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+other organization... you'll just need the permissions to do so.
+
+#. First, let's set up a new project locally using the Django Project Template:
+
+   .. code-block:: bash
+
+      $ mkvirtualenv -p `which python3` myproject
+      (myproject)$ pip install Django
+      ... Django installs ...
+      (myproject)$ django-admin.py startproject \
```
Comment (dpoirier) [inline source/services/github.rst]: This duplicates instructions in the project template, and is likely to get out of sync with it. It's also not really relevant to github - maybe we should just skip over this with "once you have your local project set up, ..." and then continue on with "git init"?

File: source/services/github.rst [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+   .. code-block:: bash
+
+      (myproject)~/myproject$ git remote add origin https://github.com/caktus/myproject.git
+      (myproject)~/myproject$ git push -u origin master
+
+#. Review the 'Settings' tab to configure the repository to your preferences:
+
+   A. Manage outside collaborators.
+   #. Link Github to other services.
```
Comment (dpoirier) [inline source/services/github.rst]: maybe add `(such as HipChat or Travis CI)`, or something more generic if you want. `services` seems a little too generic though :-)

**Comment by dpoirier** [comment]
Having the direct links to Github pages for creating repositories and managing the team seems like it'll be very convenient.

**Comment by vkurup** [comment]
Good points, thanks. This is updated.

**Comment by dpoirier** [comment]
:sunglasses:

**Pull request merged**